core/client/fs/pom.xml

```diff
@@ -39,6 +39,10 @@
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java</artifactId>
     </dependency>
+    <dependency>
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
+    </dependency>
```

Comment:
I understand the reasoning for adding this dependency, but why all of the others?

Reply:
they are listed as used but not listed in pom, so I figured we should add them? probably ok to leave them out for now to ensure minimal change.

Reply:
@ZacBlanco ping?

Reply:
Can you update the description with more information? Otherwise LGTM